packages/containers/Import/Steps/ImportingStep/ImportingStep.js

```diff
@@ -38,7 +38,7 @@ const TwoItemsWrapper = styled.div`
 `;
 
 const renderText = progress =>
-  progress < 1
+  progress < 0.99999
```

Comment:
why such a number?

Reply:
Sometimes prop `progress` has value like 0.9999999999999 when import is already finished

Reply:
o_O
`0.99` like a 99% 
// nitpick